helm/flink-operator/values.yaml

```diff
@@ -61,6 +61,7 @@ operatorConfiguration:
     metrics.reporter.slf4j.interval: 5 MINUTE
 
     operator.reconciler.reschedule.interval.sec: 15
+    operator.reconciliation.max.parallelism: -1
```

Comment:
we do not need to add this here, it's already in the defaults

Reply:
That's removed now.